src/PackageManagement/BuildIntegration/BuildIntegratedRestoreUtility.cs

```diff
@@ -73,7 +73,9 @@ public static class BuildIntegratedRestoreUtility
             request.MaxDegreeOfConcurrency = PackageManagementConstants.DefaultMaxDegreeOfParallelism;
 
             // Add the existing lock file if it exists
-            request.ExistingLockFile = GetLockFile(project, logger);
+            var lockFilePath = BuildIntegratedProjectUtility.GetLockFilePath(project.JsonConfigPath);
```

Comment:
This is just an in-memory string operation to create a lock file name based on the project.json name.